pkg/virt-controller/services/vm.go

```diff
@@ -93,14 +93,14 @@ func (v *vmService) DeleteVMPod(vm *corev1.VM) error {
 	precond.MustNotBeNil(vm)
 	precond.MustNotBeEmpty(vm.GetObjectMeta().GetName())
```

Comment:
may add a precondition for namespace here now that it is a required argument.  However, I can't think of a scenario where name would exist but namespace would not.